distributed/utils_test.py

```diff
@@ -1761,21 +1756,14 @@ def check_instances():
 @contextmanager
 def config_for_cluster_tests(**extra_config):
     "Set recommended config values for tests that create or interact with clusters."
-    reset_config()
-
-    with dask.config.set(
+    with new_config(
```

Comment:
By using `new_config`, we get `initialize_logging`. The behavior should be mostly the same (in both cases, we first reset the config to the original).